restaurant/src/app/admin/admin.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
```

Comment:
if this component is not used, remove it with its dependencies like html,css,spec.ts, ts file